src/Goerli-DssSpell.t.sol

```diff
@@ -504,4 +556,485 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+            ilk:         "RWA012-A",
+            ilkString:   "RWA012-A",
+            LIQ:         addr.addr("MIP21_LIQUIDATION_ORACLE"),
+            GEM_JOIN:    addr.addr("MCD_JOIN_RWA012_A"),
+            URN:         addr.addr("RWA012_A_URN"),
+            CEIL:        30_000_000 * WAD,
+            PRICE:       36_499_587 * WAD,
+            ROOT:        0x60b71e9DCEeDAAC275c377630E054bc60a21A02B,
+            COORDINATOR: 0x3f82851463C172DBDc1229cA06170fF89f5638dC,
+            DROP:        0x82b84166f7CB140A6a66308da10728a3DB3A73A4,
+            MEMBERLIST:  0xAe9f348Dd245BCdD7D3D6Bbb20346059B2259b71,
+            MGR:         0xaef64c80712d5959f240BE1339aa639CDFA858Ff
+        }));
+
+        collaterals.push(CentrifugeCollateralTestValues({
+            ilk:         "RWA013-A",
+            ilkString:   "RWA013-A",
+            LIQ:         addr.addr("MIP21_LIQUIDATION_ORACLE"),
+            GEM_JOIN:    addr.addr("MCD_JOIN_RWA013_A"),
+            URN:         addr.addr("RWA013_A_URN"),
+            CEIL:        70_000_000 * WAD,
+            PRICE:       85_165_703 * WAD,
+            ROOT:        0xCd5Cb76a0208eAbdFFC2074f32591878a10686ae,
+            COORDINATOR: 0x3C896EF5d7648Dd11CBDE5EDd6470063a0e780cA,
+            DROP:        0x0691FAEa2Eb8eBB2C36Fc24d577cA73AfbDB7Bdd,
+            MEMBERLIST:  0xbf78fff127d58f177531045d1E3588a03847Ac4C,
+            MGR:         0xc5A1418aC32B5f978460f1211B76B5D44e69B530
+        }));
+
+        for (uint256 i = 0; i < collaterals.length; i++) {
+            _setupCentrifugeCollateral(collaterals[i]);
+        }
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        for (uint256 i = 0; i < collaterals.length; i++) {
+            _tinlakeMgrLock(collaterals[i]);
+        }
+    }
+
+    function _setupCentrifugeCollateral(CentrifugeCollateralTestValues memory collateral) internal {
+        Root root = Root(collateral.ROOT);
+        TinlakeManagerLike mgr = TinlakeManagerLike(collateral.MGR);
+        DSTokenAbstract drop = DSTokenAbstract(collateral.DROP);
+        MemberList memberlist = MemberList(collateral.MEMBERLIST);
+
+        // Welcome to hevm KYC
+        hevm.store(collateral.ROOT, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        assertEq(root.wards(address(this)), 1);
+
+        root.relyContract(collateral.MEMBERLIST, address(this));
+
+        memberlist.updateMember(address(this), type(uint256).max);
+        memberlist.updateMember(collateral.MGR, type(uint256).max);
+
+        // Set this contract as `ward` on `mgr`
+        hevm.store(collateral.MGR, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        assertEq(mgr.wards(address(this)), 1);
+
+        // File MIP21 contracts
+        FileLike(collateral.MGR).file("liq", collateral.LIQ);
+        FileLike(collateral.MGR).file("urn", collateral.URN);
```

Comment:
These are set by the spell and should not be overriden here. You can remove these two lines.

Reply:
Done